git_machete/cmd.py

```diff
@@ -2536,31 +1864,30 @@ def load_all_reflogs(cli_ctxt: CommandLineContext) -> None:
         if len(branch_and_pos) != 2:  # invalid, shouldn't happen
             continue
         b, pos = branch_and_pos
-        if b not in reflogs_cached:
-            reflogs_cached[b] = []
-        reflogs_cached[b] += [(sha, subject)]
+        if b not in GitContext.reflogs_cached:
+            GitContext.reflogs_cached[b] = []
+        GitContext.reflogs_cached[b] += [(sha, subject)]
 
 
 def reflog(cli_ctxt: CommandLineContext, b: str) -> List[REFLOG_ENTRY]:
```

Comment:
Same here, this method probably belongs to `GitContext` as well, reflogs are by no means machete-specific